Docker gstf readme (#125)
TGAC/earlham-galaxytools: docker/GSTF-docker/readme.md

# GeneSeqToFamily Docker

The GeneSeqToFamily workflow is a comprehensive pipeline based on Ensembl Compara pipeline to find gene families. 

## Users and passwords

> The Galaxy Admin User has the username admin@galaxy.org and the password admin. In order to use certain features of Galaxy, like e.g. workflows, one has to be logged in. Also the installation of additional tools requires a login.

Reviewer: ```suggestion
The Galaxy Admin User has username `admin@galaxy.org` and password `admin`. In order to use certain features of Galaxy, for example workflows, one has to be logged in. Also the installation of additional tools requires a login.
```

> The PostgreSQL username is galaxy, the password galaxy and the database name galaxy.

Reviewer: ```suggestion
The PostgreSQL username is `galaxy`, the password `galaxy` and the database name `galaxy`.
```

> If you want to create new users, please make sure to use the /export/ volume. Otherwise your user will be removed after your docker session is finished.

Reviewer: It's not very clear how to do that.

## Starting docker

Starting the workflow is similar to start the generic Galaxy Docker image:

```
$ docker build -t <tag> . 
```

```
$ docker run -i -t -p 8080:80 <tag>
```

A detailed discussion of Docker's parameters is given in the [Docker manual](http://docs.docker.io/). It is really worth reading. Nevertheless, here is a quick rundown:

## Adding custom data

> Docker configuration comes with test data, but if a user wants to start docker with custom data then modify the `library_data.yml` file as shown in example within. These data will be available as data libraries in Galaxy.

Reviewer: ```suggestion
Docker configuration comes with test data, but if you want to pre-load custom data, then modify the `library_data.yml` file as shown in the example within. These data will be available as data libraries in Galaxy.
```
